Sources/Sentry/Public/SentryHub.h

```diff
@@ -44,7 +44,9 @@ SENTRY_NO_INIT
                  withScope:(SentryScope *)scope NS_SWIFT_NAME(capture(event:scope:));
 
 /**
- * Creates a transaction bound to the hub and returns the instance.
+ * Creates a transaction bound it to the hub and returns the instance.
+ * The created transaction is added to the scope using the name as key.
+ * A transaction already in the scope with the same name is replaced.
```

Comment:
> added to the scope using the name as key

docs are outdated, this PR does not do it anymore

Reply:
Ops, forgot to change this comment. 
My bad.